addons/macrochanger/macrochanger.lua

```diff
@@ -26,143 +26,85 @@
 
 _addon.name = 'MacroChanger'
 _addon.author = 'Banggugyangu'
-_addon.version = '1.0.0.1'
+_addon.version = '1.0.0.2r0'
 _addon.commands = {'mc','macrochanger'}
 
+config = require('config')
 require('strings')
 
-windower.register_event('load', function()
-	globaldisable = 0
-    macros = {
-        WAR = {Book = '', Page = ''},
-        MNK = {Book = '', Page = ''},
-        WHM = {Book = '', Page = ''},
-        BLM = {Book = '', Page = ''},
-        RDM = {Book = '', Page = ''},
-        THF = {Book = '', Page = ''},
-        PLD = {Book = '', Page = ''},
-        DRK = {Book = '', Page = ''},
-        BST = {Book = '', Page = ''},
-        BRD = {Book = '', Page = ''},
-        RNG = {Book = '', Page = ''},
-        SAM = {Book = '', Page = ''},
-        NIN = {Book = '', Page = ''},
-        DRG = {Book = '', Page = ''},
-        SMN = {Book = '', Page = ''},
-        BLU = {Book = '', Page = ''},
-        COR = {Book = '', Page = ''},
-        PUP = {Book = '', Page = ''},
-        DNC = {Book = '', Page = ''},
-        SCH = {Book = '', Page = ''},
-        GEO = {Book = '', Page = ''},
-        RUN = {Book = '', Page = ''},
-        }
-	options_load()
-end)
+function sleep (n)
+  local t0 = os.clock()
+  while os.clock() - t0 <= n do end
+end
```

Comment:
We have a sleep function you can use: `coroutine.sleep(2)`. That way you don't have to busy wait.